crates/ironrdp-dvc/src/pdu/tests/capabilities.rs

```diff
@@ -0,0 +1,69 @@
+use crate::vec;
```

Comment:
Let's move these tests to `ironrdp-test-suite` and add following to `Cargo.toml`

```
[lib]
doctest = false
test = false
```

Reply:
bc26ef08e3a73e2cc56dbe73e8b67103abc9f0e5